app/controllers/catalog_controller.rb

```diff
@@ -2051,7 +2051,7 @@ def replace_right_cell(options = {})
       elsif action == "dialog_provision"
         presenter.hide(:toolbar)
         # incase it was hidden for summary screen, and incase there were no records on show_list
-        presenter.show(:paging_div, :form_buttons_div).hide(:pc_div_1)
+        presenter.show(:buttons_on, :form_buttons_div).hide(:buttons_off, :paging_div, :pc_div_1)
```

Comment:
Hey @h-kataria as far as I can understand, this will "enable" the submit/cancel buttons after ordering, right? If that is the case, we **do not** want this to happen unless it is the old dialog-user ui, which should be taken care of further down during the block on line 2062.

This is because the new dialog-user component uses its own buttons and I accidentally hid them when ordering a service because of the work on custom buttons I was doing. The following PR should fix that issue https://github.com/ManageIQ/manageiq-ui-classic/pull/2753. However, if we then are showing the buttons at the bottom of the screen, then we'll have double buttons. (Not to mention the buttons on the bottom don't work for the new dialog-user component anyway)